src/about.njk

```diff
@@ -10,42 +10,42 @@ eleventyNavigation:
     <p>Il Trio FiBre ha l'obiettivo di scoprire compositori coevi e spaziare dalla Musica del '900 alla Contemporanea all' Elettronica e alla compartecipazione di arti musicali e visive nelle loro più creative possibilità.</p>
     <p>Il Trio FiBre nasce a Firenze nel 2022, i musicisti che lo compongono hanno tutti compiuto gli studi presso il Conservatorio di Firenze e hanno svolto attività concertistica da camera, sinfonica e contemporanea per importanti istituzioni.</p>
     <div class="controls-container">
-      <button class="control" disabled>
+      <button class="controls" disabled>
       {% include "./_includes/icon-components/chevron-left.njk" %}
        <span class="sr-only">Previous card</span>
       </button>
-      <button class="control">
+      <button class="controls">
         {% include "./_includes/icon-components/chevron-right.njk" %}
         <span class="sr-only">Next card</span>
       </button>
     </div>
   </div>
-  <div class="card-container">
-    <div class="card-scroll">
-      <article class="flex-card">
-        <h2> Daniela Nocentini </h2>
-        <p>Clarinet</p>
-        <a href="#" class="cta">More info</a>
-        <img src="../assets/images/daniela-card.jpg" alt="Portrait of Danilea Nocentini" width="100%">
+  <ul class="cards-container">
+    <li class="card">
+      <article class="card-content">
+      <h2>Daniela Nocentini</h2>
+      <p>Clarinet</p>
+      <a href="https://christianheilmann.com/2020/11/26/back-to-basics-creating-a-clickable-card-interface-in-plain-html-css-and-javascript/" class="cta">More info</a>
```

Comment:
Link text could be improved for accessibility

Reply:
This is just a placeholder! Normally clicking here would bring to Daniela's personal page

Reply:
Do you mean the 'More info' here? I'm on it